src/mlpack/bindings/python/print_pyx.cpp

```diff
@@ -143,10 +143,16 @@ void PrintPYX(const ProgramDoc& programInfo,
 
   // Print the comment describing the function and its parameters.
   cout << "  \"\"\"" << endl;
-  cout << "  " << programInfo.programName << endl;
+  cout << "  " << pname.programName << endl;
   cout << endl;
-  cout << "  " << HyphenateString(programInfo.documentation(), 2) << endl;
-  cout << endl << endl;
+  cout << "  " << HyphenateString(longDesc.longDescription(), 2) << endl;
+    cout << endl;
```

Comment:
```suggestion
  cout << endl;
```
Simple style fix. :)